src/theory/datatypes/sygus_datatype_utils.cpp

```diff
@@ -343,6 +351,9 @@ Node sygusToBuiltin(Node n, bool isExternal)
     it = visited.find(cur);
     if (it == visited.end())
     {
+      // notice this condition succeeds in roughly 99% of the executions of this
```

Comment:
Could you give more details on this number? For typical benchmarks, regressions...? Circa 20XX?

Reply:
Coverage tests :)  They are helpful.

Reply:
Ah, nice. :)